src/main/java/net/dv8tion/jda/api/entities/Invite.java

```diff
@@ -503,4 +589,70 @@ default String getUrl()
         GROUP,
         UNKNOWN
     }
+
+    /**
+     * Enum representing the target type of an invite.
```

Comment:
Either this or `Invite#getTargetType` need to have some elaboration on what a "TargetType" _is_.
The docs for them are essentially "The thing is the thing". Not particularly helpful.

Something like "A TargetType indicates additional action to be taken by the client on accepting the invite, typically connecting external services or launching external applications depending on the specific TargetType." would be more descriptive.